cmake/legacy.cmake

```diff
@@ -6,6 +6,43 @@
 #                  copied verbatim in the file "LICENSE"                       #
 ################################################################################
```

Comment:
Please update the copyright. I think, we already have 2024.

Reply:
Will be updated.